### DISCUSSION TITLE
Separated category for Disabled mods

### DISCUSSION THREAD

**Discussion by offron** [piotrulos/MSCModLoader#169] [General]
I have a lot of mods installed. It would be helpfull to have a separate Category for disabled Mods.

Especially if someone has to troubleshoot or just simply want to try out diffrent things .
Its helpfull to see which mods are Enabled / Disabled without having to carefully scroll trough every single Mod in the List.

Maybe in a seperate Tab? Or maybe just at the End of the Mod list?

PS: I know i could try to lessen my installed mods (which i did already), but having the ease of MSCLoader and greate Mods,
does not help to reduce the Number by much.

**Comment by piotrulos** [comment]
This is done by search filters, just search [Disabled] if you want list of all disabled mods.

Edit: wait nvm, this isn't yet part of public build
